### PR TITLE
Microsoft community main

### PR DESCRIPTION
**Pull request by MetkarAdi**

### CODE REVIEW

File: src/features/commands/buttons/doNothing.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const { Client } = require('discord.js'),
+    db = require('../../../db'),
+    perspective = require('../../../api/perspective');
```
Comment (Jasius) [inline src/features/commands/buttons/doNothing.js:3]: Unnecessary dependencies
```suggestion
const { Client } = require('discord.js');
```

File: src/features/commands/buttons/removeAll.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const {
+    Client
+} = require('discord.js'),
+    db = require('../../../db'),
+    perspective = require('../../../api/perspective');
```
Comment (Jasius) [inline src/features/commands/buttons/removeAll.js:5]: Unnecessary dependencies
```suggestion
const {
    Client
} = require('discord.js'),
    db = require('../../../db');
```

File: src/features/commands/buttons/removeAll.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        const refreshWatchedCollection = () => (
+            watchedKeywordsCollection = db.getWatchedKeywords()
+        )
+        if (interaction.isButton()) {
+            try {
+                db.removeWatchedKeyword(interaction.user.id, interaction.guild.id).then(resp => {
+                    refreshWatchedCollection()
+                })
+                interaction.update({
+                    content: `Removed all keywords being tracked.`,
```
Comment (Jasius) [inline src/features/commands/buttons/removeAll.js:24]:
```suggestion
                    content: `Tracking is now disabled, you will no longer be notified about keyword triggers.`,
```

File: src/features/commands/buttons/removeAll.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+                    components: [],
+                    ephemeral: true,
+                })
+                return
+            } catch (error) {
+                console.log(error)
+                interaction.reply({
+                    content: `Something went wrong:\n${error}`,
```
Comment (Jasius) [inline src/features/commands/buttons/removeAll.js:32]: Remove unsanitized error
```suggestion
                    content: 'Something went wrong.',
```

File: src/features/commands/selectMenus/keywords.js [hunk 5]
```diff
@@ -7,7 +7,7 @@ const {
 module.exports = {
     name: 'keywords',
     type: 'selectMenu',
-    description: 'List of keywords to choose from.',
+    description: 'Retracts from sending Toxicity embed in the channel.',
```
Comment (Jasius) [inline src/features/commands/selectMenus/keywords.js:10]: Im a bit confused, is this a mistake as it seems unrelated?

File: src/index.js [hunk 6]
```diff
@@ -112,6 +112,8 @@ client.on('interactionCreate', async interaction => {
     }
     !interaction.isButton() ? client.commands.get(interaction.commandName)?.execute(client, interaction, activeUsersCollection) : client.commands.get(interaction.customId)?.execute(client, interaction, activeUsersCollection);
     !interaction.isButton() ? collectCommandAnalytics(interaction.commandName, interaction.options?._subcommand) : collectCommandAnalytics(interaction.customId);
+    // !interaction.isSelectMenu() ? client.commands.get(interaction.commandName)?.execute(client, interaction) : client.commands.get(interaction.customId)?.execute(client, interaction);
+    // !interaction.isSelectMenu() ? collectCommandAnalytics(interaction.commandName, interaction.options?._subcommand) : collectCommandAnalytics(interaction.customId);
```
Comment (Jasius) [inline src/index.js:116]:
```suggestion
```

**Review by Jasius — CHANGES_REQUESTED** [review]

**Comment by MetkarAdi** [comment]
Made this PR by mistake, please ignore it.

**Comment by Jasius** [comment]
> Made this PR by mistake, please ignore it.

All good 👍
